### ISSUE TITLE
checkstyle : module naming error

### ISSUE THREAD

**Issue by eugenepeh** [se-edu/addressbook-level4#276]
Module names such as ConstantNameCheck is not recognised by checkstyle plugin.

**Comment by damithc** [comment]
@eugenepeh can you give a little bit more information/context about this issue?

**Comment by eugenepeh** [comment]
Hi Prof. Damith, 
I am using eclipse's checkstyle plugin to do my research work and I found that checkstyle plugin on eclipse does not recognise some of the checkstyle configuration on the xml file. 

I investigated further using http://checkstyle.sourceforge.net/ documentations as reference and found that
maybe the module name could be incorrect such as "ConstantNameCheck" should be just "ConstantName" and other module name such as "TypeNameCheck", "StaticVariableNameCheck", "MemberNameCheck" and "MethodNameCheck" too maybe incorrect as well. 

I have created a pull request to fix the above errors, group them according to documentation and including a comment to find their respective documentation.
